hare/consensus_test.go

```diff
@@ -161,11 +160,11 @@ func TestConsensusFixedOracle(t *testing.T) {
 	set1 := NewSetFromValues(value1)
 	test.fill(set1, 0, totalNodes-1)
 	test.honestSets = []*Set{set1}
-	oracle := eligibility.New()
+	oracle := &trueOracle{}
```

Comment:
why are you creating a so called `true` oracle if the method's name is `FixedOracle` test

Reply:
Fixed